extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js

```diff
@@ -74,15 +87,26 @@ require([
 	}
 
 	function experimentEnabled() {
+		experimentGroup = abTest.getGroup(experimentNames[contentLanguage]);
 		return experiments.hasOwnProperty(experimentGroup);
 	}
 
 	function setupExperiment(resources) {
-		var experiment = experiments[experimentGroup];
+		var experiment = experiments[experimentGroup],
+			templateData;
 
 		loader.processStyle(resources.styles);
+		mw.messages.set(resources.messages);
+
+		templateData = {
+			bannerType: experiment.type,
+			language: contentLanguage,
+			messageText: mw.message('pmp-entry-point-message').plain(),
+			buttonText: mw.message('pmp-entry-point-button').escaped(),
```

Comment:
`buttonText` is escaped in the template with `{{buttonText}}` so this will be double escaped. Can just be `.plain()`.